build_scripts/install_promscale.sh

```diff
@@ -20,25 +20,8 @@ export PATH="/usr/lib/postgresql/${PGVERSION}/bin:${PATH}"
 mkdir -p /home/postgres/.pgx
 
 for PROMSCALE_VERSION in "$@"; do
-    git clean -e target -f -x
-    git reset HEAD --hard
-    git checkout "${PROMSCALE_VERSION}"
-
-    MAJOR_MINOR="$(awk '/^version/ {print $3}' ./Cargo.toml | tr -d "\"" | cut -d. -f1,2)"
-    MAJOR="$(echo "${MAJOR_MINOR}" | cut -d. -f1)"
-    MINOR="$(echo "${MAJOR_MINOR}" | cut -d. -f2)"
-
-    if [ "${MAJOR}" -le 0 ] && [ "${MINOR}" -le 3 ]; then
-        cargo install cargo-pgx --version '^0.2'
-    else
-        cargo install cargo-pgx --git https://github.com/timescale/pgx --branch promscale-staging
-    fi
-    cargo pgx init "--pg${PGVERSION}" "/usr/lib/postgresql/${PGVERSION}/bin/pg_config"
-    if [ "${MAJOR}" -le 0 ] && [ "${MINOR}" -le 3 ]; then
-        PG_VER=pg${PGVERSION} make install || exit 1;
-    elif [ "${PROMSCALE_VERSION}" = "0.5.1" ]; then
-        make package && cp -v --recursive "./target/release/promscale-pg${PGVERSION}/"* / || exit 1
-    else
-        (make package && make install) || exit 1;
-    fi
+    DEBVERSION="$(apt info -a promscale-extension-postgresql-${PGVERSION} | awk '/Version:/ {print $2}' | grep "${PROMSCALE_VERSION}" | head -n 1)"
+    apt-get download promscale-extension-postgresql-${PGVERSION}=${DEBVERSION}
+    dpkg --install --admindir /tmp/dpkg --force-depends --force-not-root --force-overwrite promscale-extension-postgresql-${PGVERSION}*${DEBVERSION}*.deb
```

Comment:
Unfortunately this installation method doesn't quite do what we want. We want to have all of the versioned shared objects on disk, but installing like this doesn't do that... It's possible/probable that the fault lies with how we've packaged our extension (it seems as though toolkit has the same problem).

The issue is that

Reply:
Yes, I see only 1 version specific library exists. Going to try a few more things here though.

Reply:
I removed the the dpkg directory in between installations; that does seem to give us what we want:

- for every minor version, a separate shared object.

*toolkit*
```bash
postgres@8161a70cf214:~$ find /usr/lib -name '*toolkit*.so'
/usr/lib/postgresql/14/lib/timescaledb_toolkit-1.6.0.so
/usr/lib/postgresql/14/lib/timescaledb_toolkit-1.8.0.so
/usr/lib/postgresql/14/lib/timescaledb_toolkit-1.7.0.so
/usr/lib/postgresql/12/lib/timescaledb_toolkit-1.6.0.so
/usr/lib/postgresql/12/lib/timescaledb_toolkit-1.8.0.so
/usr/lib/postgresql/12/lib/timescaledb_toolkit-1.7.0.so
/usr/lib/postgresql/13/lib/timescaledb_toolkit-1.6.0.so
/usr/lib/postgresql/13/lib/timescaledb_toolkit-1.8.0.so
/usr/lib/postgresql/13/lib/timescaledb_toolkit-1.7.0.so
```
*promscale*
```bash
postgres@8161a70cf214:~$ find /usr/lib -name '*promscale*.so'
/usr/lib/postgresql/14/lib/promscale-0.5.2.so
/usr/lib/postgresql/14/lib/promscale-0.5.1.so
/usr/lib/postgresql/14/lib/promscale-0.5.0.so
/usr/lib/postgresql/12/lib/promscale-0.5.2.so
/usr/lib/postgresql/12/lib/promscale-0.5.1.so
/usr/lib/postgresql/12/lib/promscale-0.5.0.so
/usr/lib/postgresql/13/lib/promscale-0.5.2.so
/usr/lib/postgresql/13/lib/promscale-0.5.1.so
/usr/lib/postgresql/13/lib/promscale-0.5.0.so
```

Reply:
🤯